packages/node-integration-tests/suites/tracing/apollo-graphql/test.ts

```diff
@@ -0,0 +1,33 @@
+import { assertSentryTransaction, getEnvelopeRequest, runServer, conditionalTest } from '../../../utils';
+
+conditionalTest({ min: 12 })(() => {
```

Comment:
not supported by node 10?

Reply:
Looks like they dropped support for Node 10.
https://github.com/graphql/graphql-js/blob/main/package.json#L30

[They're using `flatMap`](https://github.com/getsentry/sentry-javascript/runs/6116666638?check_suite_focus=true#step:6:49)

Reply:
Cool, can we leave a comment above the test linking to the package.json?